scripts/react_native_pods.rb

```diff
@@ -108,21 +108,3 @@ def flipper_post_install(installer)
     end
   end
 end
-
-# Pre Install processing for Native Modules
-def codegen_pre_install(installer, options={})
-  # Path to React Native
-  prefix = options[:path] ||= "../node_modules/react-native"
-
-  # Path to react-native-codegen
-  codegen_path = options[:codegen_path] ||= "#{prefix}/../react-native-codegen"
-
-  # Handle Core Modules
-  Dir.mktmpdir do |dir|
-    native_module_spec_name = "FBReactNativeSpec"
-    schema_file = dir + "/schema-#{native_module_spec_name}.json"
-    srcs_dir = "#{prefix}/Libraries"
-    schema_generated = system("node #{codegen_path}/lib/cli/combine/combine-js-to-schema-cli.js #{schema_file} #{srcs_dir}") or raise "Could not generate Native Module schema"
-    specs_generated = system("node #{prefix}/scripts/generate-native-modules-specs-cli.js ios #{schema_file} #{srcs_dir}/#{native_module_spec_name}/#{native_module_spec_name}") or raise "Could not generate code for #{native_module_spec_name}"
-  end
-end
```

Comment:
The Podfiles no longer handle codegen. This logic has been moved to a standalone Ruby script, `scripts/generate_native_modules_specs.rb`.